### ISSUE TITLE
left-nav styling removed

### ISSUE THREAD

**Issue by rdukeshier** [PBGUX/designsystem#75]
Somewhere between 2.2.0 and 2.5.0 the left-nav and menu-collapse styling got removed.  This is not listed in the breaking changes, but it certainly broke us.

**Comment by smlombardi** [comment]
can you show me the code?  I think i know why.

did you try 2.5.2?  The leftnav scss partial is re-linked in that

**Comment by rdukeshier** [comment]
Here's how we are using it:

```
<ul class="list-unstyled menu ng-scope" spectrum-sdk-left-nav="">
  <li ng-disabled="productServiceOptionsVm.services[0].disabled"
      ng-class="{open: productServiceOptionsVm.currentService.name === 'MyService1'}" aria-disabled="false" class=""
      style=""><a class="menu-heading" href="#/MyProduct/MyService1"><i class="fa fa-plug fa-fw"></i><span
      class="menu-title ng-binding" style="display: inline-block;">My Service #1</span></a></li>
  <li ng-disabled="productServiceOptionsVm.services[1].disabled"
      ng-class="{open: productServiceOptionsVm.currentService.name === 'MyService2'}" aria-disabled="false"><a
      class="menu-heading" href="#/MyProduct/MyService2"><i class="fa fa-plug fa-fw"></i><span
      class="menu-title ng-binding" style="">My Service #2</span></a></li>
  <li ng-disabled="productServiceOptionsVm.services[2].disabled"
      ng-class="{open: productServiceOptionsVm.currentService.name === 'MyService3'}" aria-disabled="false"><a
      class="menu-heading" href="#/MyProduct/MyService3"><i class="fa fa-plug fa-fw"></i><span
      class="menu-title ng-binding" style="">My Service #3</span></a></li>
  <li ng-disabled="productServiceOptionsVm.services[3].disabled"
      ng-class="{open: productServiceOptionsVm.currentService.name === 'MyService4'}" aria-disabled="false"><a
      class="menu-heading" href="#/MyProduct/MyService4"><i class="fa fa-plug fa-fw"></i><span
      class="menu-title ng-binding" style="">My Service #4</span></a></li>
  <li class="menu-collapse"><i class="fa fa-fw"></i></li>
</ul>
```

spectrum-sdk-left-nav is essentially the pb-left-nav that was in DS.  Here's the stuff that I had to put back in to get things to look & work as they did before:

```
.menu > li.open .menu-heading i, .menu > li.open .menu-heading .menu-title, .menu > li.open .menu-heading b {
  transition: all .3s;
  color: #3e53a4; }
.minified .menu > li:hover .menu-heading {
  transition: none;
  background-color: #f8f8f8;
  width: 200px; }
.minified .menu > li:hover .menu-title {
  display: inline-block !important;
  background-color: #f8f8f8;
  font-weight: 700; }
.minified .menu > li:hover .menu-content {
  transition: all .3s;
  background-color: #f8f8f8;
  position: absolute;
  width: 165px;
  left: 45px;
  display: block !important; }
.minified .menu > li:hover .menu-content li a {
  padding: 5px; }
.menu .menu-heading {
  white-space: nowrap;
  padding: 5px 3px;
  color: #717171;
  text-decoration: none;
  display: block;
  cursor: pointer;
  position: relative; }
.menu .menu-heading i.fa {
  margin-right: 10px; }
.menu .menu-heading:hover {
  background-color: transparent;
  transition: all .3s;
  color: #3e53a4; }
.menu .menu-content {
  display: none; }
.menu .menu-content li {
  white-space: nowrap; }
.menu .menu-content li:hover {
  background: #f8f8f8;
  color: #3e53a4; }
.menu .menu-content li a {
  color: #717171;
  text-decoration: none;
  padding-left: 41px;
  padding-top: 5px;
  padding-bottom: 5px;
  display: block;
  cursor: pointer; }
.menu .menu-content li a:hover {
  color: #3e53a4; }
.menu .menu-content li a.active {
  color: #3e53a4; }
.menu .menu-collapse {
  cursor: default;
  padding-left: 3px; }
.menu .menu-collapse i.fa {
  cursor: pointer;
  transition: all .3s; }
.menu .menu-collapse i.fa:hover {
  transition: all .3s;
  color: #3e53a4; }
.menu .menu-collapse i.fa:before {
  content: '\f0a8'; }
.minified .menu .menu-collapse i.fa {
  transition: all .3s;
  -webkit-transform: rotate(180deg);
  transform: rotate(180deg); }
.no-csstransforms .minified .menu .menu-collapse i.fa:before {
  content: '\f0a9'; }
```

I installed 2.5.2 and verified it does _not_ fix the issue.  
FYI - The css header for 2.5.2 says 2.5.1 (installed via bower).

**Comment by smlombardi** [comment]
Thanks.  I think there was confusion on our end about styles for the new left navs in the split view and hamburger menu, and I suspect during a merge something happened to the older styles.  I will restore the code above to `_leftnav.scss`

**Comment by smlombardi** [comment]
Please test this and verify it's ok, if you can.
[design_system.css.zip](https://github.com/PBGUX/designsystem/files/155568/design_system.css.zip)

**Comment by rdukeshier** [comment]
That does the trick, thanks.

**Comment by smlombardi** [comment]
Fixed in 2.5.3 https://github.com/PBGUX/designsystem/commit/b1df8300310c3872af46577bb8a64a64d83f1c4b
